storm-core/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -580,20 +655,23 @@ public static void restrictPermissions(String baseDir) {
     }
 
 
-    public static synchronized IFn loadClojureFn(String namespace, String name) {
+    public static synchronized clojure.lang.IFn loadClojureFn(String namespace, String name) {
         try {
             clojure.lang.Compiler.eval(RT.readString("(require '" + namespace + ")"));
         } catch (Exception e) {
             //if playing from the repl and defining functions, file won't exist
         }
-        return (IFn) RT.var(namespace, name).deref();
+        return (clojure.lang.IFn) RT.var(namespace, name).deref();
     }
 
     public static boolean isSystemId(String id) {
         return id.startsWith("__");
     }
 
-    public static <K, V> Map<V, K> reverseMap(Map<K, V> map) {
+    /*
+        TODO: Can this be replaced with reverseMap in this file?
```

Comment:
Yep. I've made a note of this in the code and moved simpleReverseMap next to the other reverseMaps.